internal/driver/nodeserver.go

```diff
@@ -241,28 +242,28 @@ func (ns *NodeServer) NodeUnstageVolume(ctx context.Context, req *csi.NodeUnstag
 	defer done()
 
 	volumeID := req.GetVolumeId()
-	stagingTargetPath := req.GetStagingTargetPath()
-	log.V(2).Info("Processing request", "volumeID", volumeID, "stagingTargetPath", stagingTargetPath)
+	log.V(2).Info("Processing request", "volumeID", volumeID)
 
 	ns.mux.Lock()
 	defer ns.mux.Unlock()
 
 	// Validate req (NodeUnstageVolumeRequest)
 	log.V(4).Info("Validating request", "volumeID", volumeID)
-	if err := validateNodeUnstageVolumeRequest(ctx, req); err != nil {
+	err := validateNodeUnstageVolumeRequest(ctx, req)
+	if err != nil {
 		return nil, err
 	}
 
-	// Unmount the target path
-	log.V(4).Info("Unmounting and deleting staging target path", "volumeID", volumeID, "targetPath", stagingTargetPath)
-	if err := mount.CleanupMountPoint(stagingTargetPath, ns.mounter.Interface, true /* bind mount */); err != nil {
-		return nil, errInternal("Failed to unmount %s: %v", stagingTargetPath, err)
+	log.V(4).Info("Unmounting staging target path", "volumeID", volumeID, "stagingTargetPath", req.GetStagingTargetPath())
+	err = mount.CleanupMountPoint(req.GetStagingTargetPath(), ns.mounter.Interface, true /* bind mount */)
+	if err != nil {
+		return nil, errInternal("NodeUnstageVolume failed to unmount at path %s: %v", req.GetStagingTargetPath(), err)
 	}
 
 	// If LUKS volume is used, close the LUKS device
-	log.V(4).Info("Closing LUKS device", "volumeID", volumeID, "stagingTargetPath", stagingTargetPath)
-	if err := ns.closeLuksMountSource(ctx, volumeID); err != nil {
-		return nil, errInternal("Failed to close the luks volume %s: %v", volumeID, err)
+	log.V(4).Info("Closing LUKS device", "volumeID", volumeID, "stagingTargetPath", req.GetStagingTargetPath())
+	if err := ns.closeLuksMountSource(ctx, req.GetVolumeId()); err != nil {
```

Comment:
nit: we can use volumeID var here instead of `req.GetVolumeId()`

Reply:
done